api/src/pcapi/core/educational/api/dms.py

```diff
@@ -0,0 +1,165 @@
+from datetime import datetime
+from datetime import timedelta
+import logging
+import typing
+
+from pcapi import settings
+from pcapi.connectors.dms import api as dms_api
+from pcapi.connectors.dms import models as dms_models
+from pcapi.core.educational import models as educational_models
+from pcapi.core.offerers import models as offerers_models
+from pcapi.models import db
+
+
+logger = logging.getLogger(__name__)
+
+
+def update_dms_status(procedure_id: int = 0) -> None:
+    """import dms applications for eac status"""
+    if procedure_id:
+        procedures = [procedure_id]
+    else:
+        procedures = [
+            settings.DMS_EAC_PROCEDURE_INDEPENDANTS_CANDIDATE_ID,
+            settings.DMS_EAC_PROCEDURE_STRUCTURE_CANDIDATE_ID,
+            settings.DMS_EAC_PROCEDURE_MENJS_CANDIDATE_ID,
+        ]
+
+    for procedure in procedures:
+        previous_import = _get_previous_import(procedure)
+        if previous_import and previous_import.isProcessing:
+            logger.info(
+                "[DMS] Procedure %s is already being processed.",
+                procedure,
+                extra={
+                    "procedure_number": procedure,
```

Comment:
(détail) L'argument de la fonction s'appelle `procedure_id`, je propose de reprendre le même terme pour être homogène. Ou de simplement nommer `procedure` ici dans le log : 

```suggestion
                    "procedure": procedure,
```

Reply:
j'ai repris une forme de log deja utilisé ailleurs pour du DMS. Je prefererai éviter de la changer au cas ou la data fasse des stats dessus.

Reply:
la variable s'appele maintenant procedure_number ce qui est plus coherent.